quark/cli.py

```diff
@@ -124,8 +124,9 @@
 @click.option(
     "--multi-process",
     "num_of_process",
-    type=click.IntRange(min=1),
-    help="Allow analyzing APK with N processes",
+    type=click.INT,
+    help="Allow analyzing APK with N processes, where N doesn't exceeds" +
+    " the number of usable CPUs - 1, to avoid memory exhaustion.",
```

Comment:
1. The sentence is ambiguous due to the last comma. I suggest removing it to make the sentence clear. Also,
2. It doesn't mention how Quark handles an excessive number. I suggest using the term "is limited by" instead of "doesn't exceed" to indicate we will replace it with the maximum.

What do you think?

Reply:
Makes sense, will incorporate the change

Reply:
I suggest that we keep it brief in the in cli help, and in the Readme we'll define in detail for why this restriction exists.

Reply:
You are right. We should keep it brief here. Replacing that term only complicates the sentence.